scipy/_lib/_testutils.py

```diff
@@ -25,3 +26,19 @@ def deco(func):
             pass
         return dec.knownfailureif(True, msg)(func)
     return deco
+
+
+def suppressed_stdout():
```

Comment:
What's the point of this surrounding function if the decorator does not accept arguments? Unless I'm missing something, it could just be the underlying function `suppressed_stdout_decorator`, which would be used simply as `@suppressed_stdout`.